project/Dependencies.scala

```diff
@@ -65,7 +65,7 @@ object Dependencies {
     // ssl-config
     val sslConfigCore = "com.typesafe"                %% "ssl-config-core"             % sslConfigVersion // ApacheV2
 
-    val lmdb          = "org.lmdbjava"                % "lmdbjava"                     % "0.0.5" // ApacheV2, OpenLDAP Public License
+    val lmdb          = "org.lmdbjava"                % "lmdbjava"                     % "0.6.0" // ApacheV2, OpenLDAP Public License
```

Comment:
Quite a jump in version, is this safe for us?

Reply:
I believe they changed their version number pattern, still under version 0.x: http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.lmdbjava%22%20AND%20a%3A%22lmdbjava%22

Reply:
Seems fine, here some more info of the releases:
https://github.com/lmdbjava/lmdbjava/issues/60
https://github.com/lmdbjava/lmdbjava/issues/58